scripts/run_docker_build.sh

```diff
@@ -40,14 +44,11 @@ conda install conda-build-all
 conda install conda-forge-build-setup
 source run_conda_forge_build_setup
 
-# We don't need to build the example recipe.
-rm -rf /conda-recipes/example
-
 # yum installs anything from a "yum_requirements.txt" file that isn't a blank line or comment.
 find conda-recipes -mindepth 2 -maxdepth 2 -type f -name "yum_requirements.txt" \
```

Comment:
Just now realizing this, we should have renamed this one to `conda-recipes-without-example` too. However, it didn't matter much as the `example` didn't have `yum_requirements.txt`. Figured I'd mention it though.